colabs/Optymalizacja_i_rozbudowa_similarity.ipynb

```diff
@@ -0,0 +1,1044 @@
+              "      fill: #FFFFFF;\n",
+              "    }\n",
+              "  </style>\n",
+              "\n",
+              "      <script>\n",
+              "        const buttonEl =\n",
+              "          document.querySelector('#df-bcb5c7d4-fcfb-4ab5-a89e-2eeb7ad694f1 button.colab-df-convert');\n",
+              "        buttonEl.style.display =\n",
+              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
+              "\n",
+              "        async function convertToInteractive(key) {\n",
+              "          const element = document.querySelector('#df-bcb5c7d4-fcfb-4ab5-a89e-2eeb7ad694f1');\n",
+              "          const dataTable =\n",
+              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
+              "                                                     [key], {});\n",
+              "          if (!dataTable) return;\n",
+              "\n",
+              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
+              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
+              "            + ' to learn more about interactive tables.';\n",
+              "          element.innerHTML = '';\n",
+              "          dataTable['output_type'] = 'display_data';\n",
+              "          await google.colab.output.renderOutput(dataTable, element);\n",
+              "          const docLink = document.createElement('div');\n",
+              "          docLink.innerHTML = docLinkHtml;\n",
+              "          element.appendChild(docLink);\n",
+              "        }\n",
+              "      </script>\n",
+              "    </div>\n",
+              "  </div>\n",
+              "  "
+            ]
+          },
+          "metadata": {},
+          "execution_count": 7
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Liczenie rozbudowanego similarity"
+      ],
+      "metadata": {
+        "id": "gYfwjQvvaDYa"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Wybieramy aktora z testowego zbioru danych."
+      ],
+      "metadata": {
+        "id": "0ZXp-XtWaLiX"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "selected_actor = prepare_pandas_row(find_actor(train_data, 'nm0000216'))"
+      ],
+      "metadata": {
+        "id": "JY34aff26aB9"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Sprawdzamy czy metoda licząca similarity działa."
+      ],
+      "metadata": {
+        "id": "tB06ZsfXaQoP"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%time\n",
+        "ids, values = similarity_one_vs_all(train_data, selected_actor, reduced_dataset=True)"
```

Comment:
chyba powinno być similarity_one_vs_all_new

Reply:
powinno być i zostało to już zmienione. nawet osoby podobniejsze się znajdują